client/js/components/statement/assessmentTable/DpAssessmentTableCard.vue

```diff
@@ -1175,7 +1175,7 @@ export default {
     },
 
     toggleMapModal (drawingData) {
-      this.$parent.$refs.mapModal.toggleModal(drawingData)
+      this.$root.$emit('toggleModal', drawingData)
```

Comment:
I'd suggest to use a lightly more specific event name (which includes the term "drawing" pr "map").

Reply:
Makes sense, was done here [219dd87](https://github.com/demos-europe/demosplan-core/pull/825/commits/219dd876358fed66405aac467719509d21f339fa)